src/test/java/com/artipie/rpm/RpmITCase.java

```diff
@@ -48,10 +48,6 @@
  *  where SHA1 is a HEX from SHA1 of file content and TYPE is a type of file
  *  (primary, others, filelists). Don't forget to uncompress it first.
  *  repomd.xml is not compressed and stored at `repodata/repomd.xml`.
- * @todo #69:30min This test is failing on Windows build. It's failing because of error
- *  FileAlreadyExistException in PrimaryXml.close in Files.move, but it's using
- *  REPLACE_EXISTING options, so it should not fail and it's not failing on Linux.
- *  Let's discover the problem, fix it, and remove DisabledOnOs(OS.WINDOWS) annotation.
```

Comment:
@victornoel it seems you forgot to remove `@DisabledOnOs(OS.WINDOWS)` annotation

Reply:
@g4s8 arg, that's why it worked ^^